wordpress/wp-content/plugins/memberful-wp/src/metabox.php

```diff
@@ -257,3 +321,41 @@ protected function meta_field() {
     return 'memberful_'.$this->_entity;
   }
 }
+
+class Memberful_Term_ACL extends Memberful_Post_ACL {
```

Comment:
@northerner I have a feeling that `Memberful_Term_ACL` shouldn't extend `Memberful_Post_ACL`. Instead of that we should probably create a base ACL class and both, `Memberful_Post_ACL` and `Memberful_Term_ACL` should extend it.

However, this can wait until we solve the restriction issues.

Reply:
@northerner Do you have any thoughts on this? I feel strongly that we should refactor this before we merge this PR in.

Reply:
I'm working on a PR for this. Will share it soon.

Reply:
https://github.com/memberful/memberful-wp/pull/298